<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="advanceButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Advance</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>381</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_0_0"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_0_1"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_0_2"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_0_3"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_0_4"/>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_0_5"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_0_6"/>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_0_7"/>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_0_8"/>
     </item>
     <item row="0" column="9">
      <widget class="QLabel" name="label_0_9"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_1_0"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_1_1"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_1_2"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_1_3"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_1_4"/>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_1_5"/>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_1_6"/>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="label_1_7"/>
     </item>
     <item row="1" column="8">
      <widget class="QLabel" name="label_1_8"/>
     </item>
     <item row="1" column="9">
      <widget class="QLabel" name="label_1_9"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2_0"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2_1"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_2_2"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_2_3"/>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_2_4"/>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_2_5"/>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_2_6"/>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="label_2_7"/>
     </item>
     <item row="2" column="8">
      <widget class="QLabel" name="label_2_8"/>
     </item>
     <item row="2" column="9">
      <widget class="QLabel" name="label_2_9"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3_0"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_3_1"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_3_2"/>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_3_3"/>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_3_4"/>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="label_3_5"/>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label_3_6"/>
     </item>
     <item row="3" column="7">
      <widget class="QLabel" name="label_3_7"/>
     </item>
     <item row="3" column="8">
      <widget class="QLabel" name="label_3_8"/>
     </item>
     <item row="3" column="9">
      <widget class="QLabel" name="label_3_9"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4_0"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_4_1"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_4_2"/>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_4_3"/>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_4_4"/>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="label_4_5"/>
     </item>
     <item row="4" column="6">
      <widget class="QLabel" name="label_4_6"/>
     </item>
     <item row="4" column="7">
      <widget class="QLabel" name="label_4_7"/>
     </item>
     <item row="4" column="8">
      <widget class="QLabel" name="label_4_8"/>
     </item>
     <item row="4" column="9">
      <widget class="QLabel" name="label_4_9"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5_0"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_5_1"/>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_5_2"/>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_5_3"/>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_5_4"/>
     </item>
     <item row="5" column="5">
      <widget class="QLabel" name="label_5_5"/>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="label_5_6"/>
     </item>
     <item row="5" column="7">
      <widget class="QLabel" name="label_5_7"/>
     </item>
     <item row="5" column="8">
      <widget class="QLabel" name="label_5_8"/>
     </item>
     <item row="5" column="9">
      <widget class="QLabel" name="label_5_9"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6_0"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_6_1"/>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_6_2"/>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="label_6_3"/>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="label_6_4"/>
     </item>
     <item row="6" column="5">
      <widget class="QLabel" name="label_6_5"/>
     </item>
     <item row="6" column="6">
      <widget class="QLabel" name="label_6_6"/>
     </item>
     <item row="6" column="7">
      <widget class="QLabel" name="label_6_7"/>
     </item>
     <item row="6" column="8">
      <widget class="QLabel" name="label_6_8"/>
     </item>
     <item row="6" column="9">
      <widget class="QLabel" name="label_6_9"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7_0"/>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_7_1"/>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_7_2"/>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="label_7_3"/>
     </item>
     <item row="7" column="4">
      <widget class="QLabel" name="label_7_4"/>
     </item>
     <item row="7" column="5">
      <widget class="QLabel" name="label_7_5"/>
     </item>
     <item row="7" column="6">
      <widget class="QLabel" name="label_7_6"/>
     </item>
     <item row="7" column="7">
      <widget class="QLabel" name="label_7_7"/>
     </item>
     <item row="7" column="8">
      <widget class="QLabel" name="label_7_8"/>
     </item>
     <item row="7" column="9">
      <widget class="QLabel" name="label_7_9"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8_0"/>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_8_1"/>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_8_2"/>
     </item>
     <item row="8" column="3">
      <widget class="QLabel" name="label_8_3"/>
     </item>
     <item row="8" column="4">
      <widget class="QLabel" name="label_8_4"/>
     </item>
     <item row="8" column="5">
      <widget class="QLabel" name="label_8_5"/>
     </item>
     <item row="8" column="6">
      <widget class="QLabel" name="label_8_6"/>
     </item>
     <item row="8" column="7">
      <widget class="QLabel" name="label_8_7"/>
     </item>
     <item row="8" column="8">
      <widget class="QLabel" name="label_8_8"/>
     </item>
     <item row="8" column="9">
      <widget class="QLabel" name="label_8_9"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9_0"/>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_9_1"/>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="label_9_2"/>
     </item>
     <item row="9" column="3">
      <widget class="QLabel" name="label_9_3"/>
     </item>
     <item row="9" column="4">
      <widget class="QLabel" name="label_9_4"/>
     </item>
     <item row="9" column="5">
      <widget class="QLabel" name="label_9_5"/>
     </item>
     <item row="9" column="6">
      <widget class="QLabel" name="label_9_6"/>
     </item>
     <item row="9" column="7">
      <widget class="QLabel" name="label_9_7"/>
     </item>
     <item row="9" column="8">
      <widget class="QLabel" name="label_9_8"/>
     </item>
     <item row="9" column="9">
      <widget class="QLabel" name="label_9_9"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
